third_party/xla/xla/service/gpu/matmul_utils.h

```diff
@@ -45,8 +31,9 @@ limitations under the License.
 #include "xla/stream_executor/blas.h"
 #include "xla/types.h"
 #include "xla/xla_data.pb.h"
-
->>>>>>> upstream/master:third_party/xla/xla/service/gpu/matmul_utils.h
+#if GOOGLE_CUDA || TENSORFLOW_USE_ROCM
+#include "rocm/rocm_config.h"
+#endif
```

Comment:
this part should be align with upstream's

https://github.com/openxla/xla/blob/main/xla/service/gpu/matmul_utils.h#L39-L45